SteamcLog/Classes/SteamcLog.swift

```diff
@@ -10,6 +10,14 @@ import Foundation
 import Sentry
 import XCGLogger
 
+public struct SteamcLogUser {
```

Comment:
Couple overlapping things I don't like about this:
- Don't love prefixing classes with the name of the module, if this is gonna exist it should just be called "User" or "UserInfo" or something.
- A structure just to hold the one string to call one API and that is never stored seems like overkill. Seems like we could just have the API be `setUser(userId: String?)`. I can sort of see the argument for having a container type IF there were a bunch of extra properties in there we were using, but just having one string to ID the user seems fine for the foreseeable future, and it's easy to add a container type later (and just have the single string API become a convenience wrapper) if we do ever want to do that.

Reply:
@nbrooke feedback addressed.